### ISSUE TITLE
In Dark mode text is not visible properly.

### ISSUE THREAD

**Issue by DivyanshiNarang** [narayan954/dummygram#566]
### Description

"Write a caption is not visible in dark mode."

@narayan954 *Please assign this issue to me under gssoc23*

### Steps to reproduce

We can change the color of the text to white.

### Screenshots

In dark mode:
<img width="532" alt="Screenshot 2023-06-25 005804" src="https://github.com/narayan954/dummygram/assets/74779601/c772482e-2fe1-4edb-b4a0-6999358291e0">

In light mode:
<img width="523" alt="Screenshot 2023-06-25 005823" src="https://github.com/narayan954/dummygram/assets/74779601/61f9b973-e617-454b-8efe-d51bff41070c">

**Comment by narayan954** [comment]
> ### Description
> "Write a caption is not visible in dark mode."
> 
> @narayan954 _Please assign this issue to me under gssoc23_
> 
> ### Steps to reproduce
> We can change the color of the text to white.
> 
> ### Screenshots
> In dark mode: <img alt="Screenshot 2023-06-25 005804" width="532" src="https://user-images.githubusercontent.com/74779601/248508374-c772482e-2fe1-4edb-b4a0-6999358291e0.png">
> 
> In light mode: <img alt="Screenshot 2023-06-25 005823" width="523" src="https://user-images.githubusercontent.com/74779601/248508382-61f9b973-e617-454b-8efe-d51bff41070c.png">
> 
> ### Desktop (please complete the following information)
> _No response_
> 
> ### Smartphone (please complete the following information)
> _No response_
> 
> ### Additional information
> _No response_

yes we need to use variables instead of direct color for this purpose. But perhaps this'd be a one line change and I've been asked to not allow single line issues. So can you please add atleast one simple issue to this ? I'd suggest working on this issue only but for other components of the app as well such as the post content is not visible as well due to this same reason. Can you please fix all of these similar issues?
